packages/cli/src/commands/env/index.ts

```diff
@@ -126,6 +140,22 @@ export default async function main(client: Client) {
   const subArgs = argv._.slice(1);
   const { subcommand, args } = getSubcommand(subArgs, COMMAND_CONFIG);
   const { output, config } = client;
+
+  const environmentArg = argv['--environment']?.toLowerCase();
+
+  const target = environmentArg
+    ? parseEnvironmentTarget(environmentArg)
+    : ProjectEnvTarget.Development;
+
+  if (!target) {
```

Comment:
Can this ever be undefined? The development value is the fallback above

Reply:
It only falls back to `.Development` if the user doesn't pass in environment argument. If they do, it runs `parseEnvironmentTarget`, which is undefined if the argument is invalid.

Reply:
I think this should be `if (!isValidEnvTarget(target))` so you don't check validity twice. You can simplify this to something like this:

```js
  const target = argv['--environment'] || 'development';
  if (!isValidEnvTarget(target) {
    output.error(...);
    return 1;
  }
  // continue with happy path
```